.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
```

Comment:
try adding
```suggestion
env:
    PYTHONPATH: .
python:
```

Reply:
Damn, no luck. Thank you for the suggestion!